integration_tests/conftest.py

```diff
@@ -1,40 +1,67 @@
+import signal
```

Comment:
Also, I see a few formatting issues.

Can you please run the pre-commit hook mentioned here: https://github.com/sansyrox/robyn#%EF%B8%8F-to-develop-locally